plugins/gulp/gulp.plugin.zsh

```diff
@@ -20,7 +20,7 @@
 # in the current directory.
 #
 function $$gulp_completion {
-    compls="$(grep -Eo "gulp.task\((['\"](([a-zA-Z0-9]|-)*)['\"],)" gulpfile.js 2>/dev/null | grep -Eo "['\"](([a-zA-Z0-9]|-)*)['\"]" | sed s/"['\"]"//g | sort)"
+    compls=$(gulp --tasks-simple >/dev/null | sort)
 
     completions=(${=compls})
```

Comment:
`$compls` and `$completions` should be `local`, otherwise they leak into the global namespace.